experiments/test_helpers.rb

```diff
@@ -69,18 +71,23 @@ def run
       def setup_services
         puts "Creating mock service..."
         @services = []
+        mean_latency = 0.15
+        # Provide barely enough threads for us to handle the expected load (using Little's law)
+        max_threads_per_service = ((@requests_per_second / @service_count) * (0.75 * mean_latency)).to_i
         @service_count.times do
           @services << MockService.new(
-            endpoints_count: 50,
-            min_latency: 0.01,
-            max_latency: 0.3,
+            endpoints_count: 10,
+            min_latency: 0.1,
+            max_latency: 10,
             distribution: {
               type: :log_normal,
-              mean: 0.15,
-              std_dev: 0.05,
+              mean: mean_latency,
+              std_dev: 3,
```

Comment:
the standard deviation was too low, and all endpoints were ending up pretty much having the same latency